### ISSUE TITLE
Document WCAG 2.2 Standard Implementation Documentation

### ISSUE THREAD

**Issue by ozdentarikcan** [bounswe/bounswe2024group1#580]
# 📚 Documentation
## 📝 Description
Document WCAG 2.2 standard implementation documentation.
It should have the following sections:
* WCAG 2.2 Overview
* Task Overview
* Accessibility Improvements
  * Adding Alt Texts to Non-Text Content
  * Adjusting Color Contrast
  *  Adding Tabbability to the UI
  * Adding Accessibility Tests

## 👁‍🗨 Reviewer
@asligook

**Comment by ozdentarikcan** [comment]
Documentation is written [here](https://github.com/bounswe/bounswe2024group1/wiki/WCAG-2.2-Implementation-Documentation).
